WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js

```diff
@@ -183,7 +183,6 @@ function(child, childHtmlElement) {
 
 ZaLoginDialog.prototype._loginSelListener =
 function() {
-	this.setCursor("wait");
```

Comment:
Don't we need this?

Reply:
We are already showing `Loading...` text. Also whenever the request was failing, it was going in an infinite state. I think its okay to remove it.